README.adoc

```diff
@@ -235,7 +235,7 @@ http://sass-lang.com[Sass] and http://compass-style.org[Compass] are the preferr
 * http://sass-lang.com/docs/yardoc/file.SASS_REFERENCE.html[Sass Reference]
 * http://compass-style.org/reference/compass[Compass Reference]
 
-Source files are indented using 2 spaces. Haml files require the same indentation size throughout. Ruby hashes should have no leading or trailing spaces and options such be seperated by a comma followed by a space (e.g., `{:href=>'http://arquillian.org', title=>'Arquillian Project Site'}`).
+Source files are indented using 2 spaces. Haml files require the same indentation size throughout. Ruby hashes should have no leading or trailing spaces and options such be separated by a comma followed by a space (e.g., `{:href=>'http://arquillian.org', title=>'Arquillian Project Site'}`).
```

Comment:
I think it should also have "should" instead of "such" at the end of that sentence.

```suggestion
Source files are indented using 2 spaces. Haml files require the same indentation size throughout. Ruby hashes should have no leading or trailing spaces and options should be separated by a comma followed by a space (e.g., `{:href=>'http://arquillian.org', title=>'Arquillian Project Site'}`).
```